lae_site/handlers/submit_subscription.py

```diff
@@ -137,6 +137,12 @@ def render(self, request):
         # render_POST is handled by the HandlerBase parent which calls this this method after logging
         # the request.
 
+        if request.method != 'POST':
+            tmpl = env.get_template('s4-subscription-form.html')
+            msg = ("Your browser requested this page with a GET request but we expected a POST."
+                   "\n\nPlease try again.")
```

Comment:
I see there is some test coverage for this code.  It leans very heavily on mocks and is pretty light on documentation.  The consequences of which are probably that the tests present a high barrier to maintenance and that they may only catch the most superficial of problems.  Given that, I can see the pressure to not bother updating them.

Perhaps this simple change would be a good opportunity to replace the existing tests with some new, more valuable ones.

Note that the existing tests fail with this change because they use a very primitive test double of `Request` which has no `method` attribute (this is an example of how the current tests present a high barrier to maintenance).

One way or another, it seems like the failing tests should be considered a blocker to merging this change, though.  As far as I can tell, the test suite is green on master@HEAD and red for this branch.